code/game/objects/objs.dm

```diff
@@ -67,6 +67,10 @@ var/global/list/reagents_to_log = list(FUEL, PLASMA, PACID, SACID, AMUTATIONTOXI
 
 
 /obj/attackby(obj/item/weapon/W, mob/user)
+	if(integratedpai)
```

Comment:
Put this check after the `if(can_take_pai && istype(W, /obj/item/device/paicard))` check.
Otherwise you'll override all `attackby()` behavior for objects that have pAIs loaded into them.